packages/gestalt-codemods/next/toast-replace-color-variant.js

```diff
@@ -0,0 +1,130 @@
+/*
+ * Converts
+ *  <Toast color="red" /> to <Toast variant="error" />
+ *  <Toast color="white" /> to <Toast variant="default" />
+ *  <Toast color={color} /> to console.log for manual change />
+ */
+
+// yarn codemod --parser=flow -t=packages/gestalt-codemods/next/toast-replace-color-variant.js relative/path/to/your/code
+
+function getNewAttr({ attr, j, previousValue }) {
```

Comment:
Not extracting this to a general purpose helper yet, but thinking along those lines